packages/nimble-components/src/table-column/base/types.ts

```diff
@@ -44,19 +44,18 @@ export type TableColumnSortOperation =
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface TableColumnValidity extends ValidityObject {}
 
-const groupIconSize = 16;
-const sortIconSize = 16;
-const spacing = 8;
-const menuDropdownSize = 24;
+export const columnIconSize = 16; // `iconSize` token
+export const columnSpacing = 8; // `mediumPadding` token
+const menuDropdownSize = 24; // `controlSlimHeight` token
```

Comment:
I'm a tad nervous about exporting all these in-case folks start depending on these unexpectedly. The defaultMinPixelWidth calculated like this has been a bit of a workaround / hack.

Can we instead export something like a `columnHiddenIndicatorsWidth` constant and move the calculation in here instead of exporting all the partial aliases to exisiting tokens?